types/office-js-preview/index.d.ts

```diff
@@ -97396,29 +98351,29 @@ declare namespace Word {
      */
     interface ParagraphAddedEventArgs {
         /**
-         * Gets the IDs of the involved paragraphs.
+         * The source of the event. It can be local or remote (through coauthoring).
          *
          * @remarks
          * [Api set: WordApi BETA (PREVIEW ONLY)]
          * @beta
          */
-        ids: string[];
+        source: Word.EventSource | "Local" | "Remote";
         /**
-         * The source of the event. It can be local or remote (through coauthoring).
+         * The event type. See Word.EventType for details.
          *
          * @remarks
          * [Api set: WordApi BETA (PREVIEW ONLY)]
          * @beta
          */
-        source: Word.EventSource | "Local" | "Remote";
+        type: Word.EventType | "ContentControlDeleted" | "ContentControlSelectionChanged" | "ContentControlDataChanged" | "ContentControlAdded" | "CommentDeleted" | "CommentSelected" | "CommentDeselected" | "CommentChanged" | "CommentAdded" | "ContentControlEntered" | "ContentControlExited" | "ParagraphAdded" | "ParagraphChanged" | "ParagraphDeleted" | "AnnotationClicked" | "AnnotationHovered" | "AnnotationInserted" | "AnnotationRemoved";
         /**
-         * The event type. See Word.EventType for details.
+         * Gets the unique IDs of the involved paragraphs.
          *
          * @remarks
          * [Api set: WordApi BETA (PREVIEW ONLY)]
          * @beta
          */
-        type: Word.EventType | "ContentControlDeleted" | "ContentControlSelectionChanged" | "ContentControlDataChanged" | "ContentControlAdded" | "CommentDeleted" | "CommentSelected" | "CommentDeselected" | "CommentChanged" | "CommentAdded" | "ContentControlEntered" | "ContentControlExited" | "ParagraphAdded" | "ParagraphChanged" | "ParagraphDeleted";
+        uniqueLocalIds: string[];
```

Comment:
Is the point of saying "Local" that when two people are editing the same document, the paragraph ID for a given paragraph won't be the same for both users?

Reply:
Updated description to clarify